src/utils/urlPaths.ts

```diff
@@ -58,7 +59,8 @@ export enum UrlPathTemplate {
   RegistrationLandingPage = '/registration/:identifier',
   RegistrationNew = '/registration',
   RegistrationWizard = '/registration/:identifier/edit',
-  ResearchProfile = '/research-profile',
+  ResearchProfile = '/research-profile/:identifier',
+  ResearchProfileRoot = '/research-profile',
```

Comment:
Gulper litt av å måtte legge inn en ekstra verdi for root her, men ser ikke noen bedre løsning :/